doc/news/changes/minor/20170711VishalBoddu

```diff
@@ -0,0 +1,4 @@
+New: SolverFIRE implements FIRE (Fast Inertial Relaxation Engine) for solving
```

Comment:
Also: I think this deserves to live the in section for *major* changes section. Please move it there!